unit_testing/test_labels.py

```diff
@@ -165,29 +166,28 @@ def test_compute_mse_no_label_warning(caplog):
     assert 'Label mismatch' not in caplog.text
 
 
-@pytest.mark.skip(reason="Too long to run on large image!")
-@pytest.mark.parametrize("test_image", test_images)
+@pytest.mark.parametrize("test_image", test_labels)
 def test_remove_missing_labels(test_image):
     src = test_image.copy()
     ref = test_image.copy()
 
-    expected = test_image.copy()
-
     # introduce 2 discrepancies
     src.data[0, 0, 0] = 99
     src.data[0, 1, 2] = 99
 
     ref.data[0, 0, 0] = 1
     ref.data[0, 1, 2] = 1
 
-    # manually set expected
-    expected.data[0, 0, 0] = 0
-    expected.data[0, 1, 2] = 0
-
     res = sct_labels.remove_missing_labels(src, ref)
-    diff = res.data == expected.data
 
-    assert diff.all()
```

Comment:
Removing this is what fixes the slow test, right? Because it was unnecessarily comparing the full images?

Reply:
Changing the input file is what fixes the slowness (was using an image instead of a labels file). See commit message.